src/components/Cards/LikesCard/LikesCard.jsx

```diff
@@ -0,0 +1,82 @@
+// import "./VideoCard.css"
+import { HiDotsVertical } from "react-icons/hi"
+import { BsFillEyeFill, BsCheckCircleFill } from "react-icons/bs"
+import { AiFillClockCircle } from "react-icons/ai"
+import { MdPlaylistAdd, MdRemoveCircle } from "react-icons/md"
+import { useState } from "react"
+import { useLike } from "../../../contexts"
+
+export const LikesCard = ({ video }) => {
+    const { thumbnail, title, description, creatorAvatar, creator
+        , views } = video;
+
+    const { removeFromLike } = useLike();
+
+    // Shorting title and description here
+    const shortDescr = description.split(" ").slice(0, 20).join(" ");
+    const shortTitle = title.split(" ").slice(0, 8).join(" ");
```

Comment:
you can create a function to short strings

Reply:
Yeah sure, Thanks for the suggestion.